Clinical-Genomics/scout
# Display gene panel failure info in flash message instead of terminal

fix #1755

The current situation is that whenever there is a gene-related error in gene panel creation, the user gets a general error message like this one:
 
![image](https://user-images.githubusercontent.com/28093618/94421340-b1c9c280-0185-11eb-9418-f470c34bd0fc.png)

And a a more specific error gets printed to log (terminal) instead, and the user doesn't directly know what is wrong with the panel unless he/she asks someone who has access to the log.

![image](https://user-images.githubusercontent.com/28093618/94421483-e8074200-0185-11eb-9395-46bade20011b.png)

This PR changes the behavior to show a self-explanatory message to the user instead:

![image](https://user-images.githubusercontent.com/28093618/94422116-dbcfb480-0186-11eb-8c20-1e4cb5fb1449.png)


**How to test**:
1. On a local instance of Scout, modify the test panel with some non-existing genes (or for instance those from #1755) and try to create a new panel with that file.
1. See the difference between master and this branch

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] tests executed by EM, CR
